core/src/trezor/sdcard.py

```diff
@@ -1,13 +1,13 @@
 from trezorio import fatfs, sdcard
 
 if False:
-    from typing import Any, Callable, Optional, TypeVar
+    from typing import Any, Callable, TypeVar
 
     T = TypeVar("T", bound=Callable)
 
 
 class FilesystemWrapper:
-    _INSTANCE = None  # type: Optional[FilesystemWrapper]
+    _INSTANCE = None  # type: FilesystemWrapper | None
```

Comment:
this should be a new-style type declaration

Reply:
Fixed in 6f68820:
```python
_INSTANCE: "FilesystemWrapper" | None = None
```